lib/perfdata/influxdbwriter.ti

```diff
@@ -75,6 +75,9 @@ class InfluxdbWriter : ConfigObject
 	[config] bool enable_ha {
 		default {{{ return false; }}}
 	};
+
+	[state, no_user_modify] double last_failover;
```

Comment:
When the instance has been successfully connected with InfluxDB.

Reply:
And in IDO context?

Reply:
When the instance is not able to write to the ido because it is being performed by another instance.

Reply:
No. Failover means an instance _starts writing_ to the IDO because it notices that another instance _is not able to write_. This is being detected via the database. InfluxdbWriters _don’t_ communicate over the database – drop that last "failover".

https://github.com/Icinga/icinga2/blob/1b9f161aea8861e31c2744c7493d5e1c984c951e/lib/db_ido_pgsql/idopgsqlconnection.cpp#L315-L343